test/wasmfs/wasmfs_cwd.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+int main() {
```

Comment:
This test does not use the JS API either. To test the JS API, you'll need to call it from JS inside an `EM_ASM` block, not from C/C++. As an example, see [wasmfs_readfile.c](https://github.com/emscripten-core/emscripten/blob/main/test/wasmfs/wasmfs_readfile.c).